app/views/works/_related_objects_table.html.erb

```diff
@@ -4,14 +4,22 @@
 </div>
 
 <div class="related-objects">
-  <table id="related-objects-table">
-    <tbody id="related-objects-table-body">
+  <table>
+    <thead>
       <tr class="header-row">
         <th>Related Identifier (Example: URI for a related publication or resource)</th>
         <th>Related Identifier Type</th>
         <th>Relation Type</th>
       </tr>
-    </tbody>
+    </thead>
+    <tbody id="related-objects-table"></tbody>
+    <tfoot>
+      <tr>
+        <td colspan="3">
+          <%= link_to "Add Another Related Object", "#", id: "btn-add-related-object", class: "btn btn-secondary" %>
```

Comment:
Is there code that should be deleted from Related?  Maybe this needs a rebase??

Reply:
I'm not sure that I understand: The button that's added just above was moved/deleted from farther down in the file.

Reply:
The PR seems to indicate that this is a refactor.  This seems to add new functionality.

Reply:
I don't understand your comment: If we're talking about this line, the `Add Another Related Object` is being moved from a div underneath the table to a tfoot inside the table (which corresponds to the title on the PR).

Let me know what I'm missing / what needs to change?